elixir/apps/api/lib/api/gateway/channel.ex

```diff
@@ -131,6 +131,7 @@ defmodule API.Gateway.Channel do
       } do
       :ok = Flows.subscribe_to_flow_expiration_events(flow_id)
 
+      client = Clients.fetch_client_by_id!(client_id, preload: [:actor])
```

Comment:
```suggestion
      client = Clients.fetch_client_by_id!(client_id)
```

I don't think you need the send the associated actor to the Gateway right?

Reply:
I have no ides what this does, I just copied the exact thing from `request_connection` :)